### PR TITLE
feat(compounds): Add tests for inputs

### PR DESCRIPTION
**Pull request by umahaea**
Compounds:
- Add tests for inputs
- Add all supporting files for this new tests.
- Change buttons and icons customEvent to InitComponents

@JasonLantz @StommePoes @mhomolak @wyseguyonline @udhadpa for your review ..

### CODE REVIEW

File: src/test/java/compoundsTests/InputsTest.java [hunk 1]
```diff
@@ -0,0 +1,531 @@
...
+                {inputsPgObj.slUnderLineInputError, "transition-property", new String[]{"all"}},
+                {inputsPgObj.slUnderLineInputError, "transition-timing-function", new String[]{"ease", "cubic-bezier(0.25, 0.1, 0.25, 1)"}}
+        };
+    }
+
+    //Fancy - Inputs (single line - error)
+    @Test(testName = "Fancy - Verify Single Line Input - Errored", dataProvider = "Fancy - Single Line Input Errored Test Data", groups = "desktop-regression")
+    private void fancySingleLineInputErroredTest(By element, String cssProperty, String[] expectedCSSValue) throws Exception {
+        if (cssProperty.equals("transition") && browser.equals("edge")) {
+
```
Comment (udhadpa) [inline src/test/java/compoundsTests/InputsTest.java]: Log.info statement is missing

File: src/test/java/compoundsTests/InputsTest.java [hunk 2]
```diff
@@ -0,0 +1,531 @@
...
+    @Test(testName = "Fancy - Verify Single Line Input", dataProvider = "Single Line Input Test Data", groups = {"desktop-ci", "desktop-regression"})
+    private void fancySingleLineInputTest(String cssProperty, String[] expectedCSSValue) throws Exception {
+        String[] detailsPropertiesList = new String[]{"elementId", "inputs-target", "componentName", "TextInput"};
+        String[] propsPropertiesList = new String[]{"id", "sl-text-input", "label", "Last Name", "inputType", "default", "placeholder", "Last Name"};
+        testConfig = buildJSONObjectDetailConfig(detailsPropertiesList, propsPropertiesList);
+        commonUtils.changeConfig(inputsJSFilePath, testConfig);
+        commonUtils.getUrl(inputsUrl);
+        cssPropertyType = cssProperty;
+        cssProperty = commonUtils.getCSSValue(inputsPgObj.slTextInput, cssProperty);
+        isCSSProperty = commonUtils.assertCSSProperties(cssProperty.toString(), cssProperty, expectedCSSValue);
```
Comment (udhadpa) [inline src/test/java/compoundsTests/InputsTest.java]: Why do we have to do cssProperty.toString() ? Isn't cssProperty of type String?

Comment (umahaea) [inline src/test/java/compoundsTests/InputsTest.java]: u r right payal..I will remove it.. don't remember why I did that.. :) I will fix it.

File: src/test/java/compoundsTests/InputsTest.java (hunk 1 above)
Comment (umahaea) [inline src/test/java/compoundsTests/InputsTest.java]: Good catch.. The condition is no more valid. I probably used it for debugging. I will remove it.

**Comment by udhadpa** [comment]
Rest looks good. Great work Eajaz.

**Comment by umahaea** [comment]
Got enough approvals. merging it.

**Pull request merged**